src/renderer/atlas/AtlasEngine.cpp

```diff
@@ -788,8 +788,6 @@ void AtlasEngine::_createSwapChain()
 
         if (_api.hwnd)
         {
-            desc.AlphaMode = DXGI_ALPHA_MODE_IGNORE;
-

```

Comment:
This one is redundant with a ternary statement a few lines above, where we already set the correct `AlphaMode` if a HWND is present.